### DISCUSSION TITLE
Load a LoRA in Replacer like Img2Img

### DISCUSSION THREAD

**Discussion by ruichang555** [light-and-ray/sd-webui-replacer#101] [Q&A]
Can you use lora in replacer (not in the hairfix), but load a LoRa model in the text prompt like Img2Img in WebUI?

**Comment by light-and-ray** [comment]
You can copy <lora...> in replacers prompt and it will work. Im bot planning to add Lora tab from img2img/txt2txt because it's hard and heavy (will slow down ui)

**Reply by ruichang555** [reply]
Perfect. Another question on Replacer-video. Are you planning to add clothes swap function in there? I am trying to change clothes in a video but there are no good XL checkpoint and CN models for that purpose. Appreciate your further thoughts on this

**Reply by light-and-ray** [reply]
> Perfect. Another question on Replacer-video. Are you planning to add clothes swap function in there? I am trying to change clothes in a video but there are no good XL checkpoint and CN models for that purpose. Appreciate your further thoughts on this

I don't understand. Do you mean any special clothing model? I don't know such model, I remember there are such models for images, but bot for video. Video tab now supports only animatediff. For sdxl I agree there are no inpainting model. Instead of controlnet union promaxz but a special preprocessor is needed, I ve asked for it here   https://github.com/Mikubill/sd-webui-controlnet/issues/3035 I couldn't implement this preprocessor
